index.html

```diff
@@ -44,8 +44,8 @@
                 <h3>Hello! Sharing World</h3>
                 <h1>Haitian Otaku,<br> Global Summit<br>2023</h1>
                 <p class="heroDesc"> Lorem ipsum, dolor sit amet consectetur adipisicing elit. Accusantium dolor a quod asperiores fugit iure facere ipsum cumque est? Eveniet perspiciatis molestias explicabo, aliquid aperiam temporibus quasi velit dolorum odio?</p>
-                <h2>2023.08.30(Wed) - 31(Thu) </h2>
-                <p>Fubar Club of Port-au-Prince, Art Center Nabi and more</p>
+                <h2>2023.08.30(Wed) ~ 31(Thu) </h2>
+                <p class ="adress">Fubar Club of Port-au-Prince, Art Center Nabi and more</p>
             </div>
         </section>
```

Comment:
- [ ] As mentioned in the previous reviews, It will be nice to have all the elements aligned on the same starting point as shown in the images below.

> As mentioned by the previous reviewer, it's essential to strive for a high degree of alignment with the design. They also emphasized the importance of adding more margins to your text elements within the main section and aligning them consistently. However, upon closer inspection, there are still some adjustments needed to achieve a closer match to the design.
> 
> * [ ]  One notable observation is that all the elements start from the left side and maintain the same margins. To better align with the design, it's crucial to ensure that each element's placement and margins closely mirror the intended layout. This attention to detail will contribute to a polished and visually cohesive presentation.
> 
> Your	Design
> ![image](https://user-images.githubusercontent.com/55840999/259911590-66c73d84-7fb9-4169-9492-c2c6c847f9d6.png)	![image](https://user-images.githubusercontent.com/55840999/259911650-69e298a0-40f6-4121-824f-9c36be4384cf.png)
> Keep up the great work and continue fine-tuning these elements to achieve seamless integration between your project and the design. Your dedication to these refinements will surely elevate the overall quality of your work! star2memorocket